admin_manual/ai/app_context_chat.rst

```diff
@@ -149,64 +149,11 @@ Configuration Options (OCC)
 
    occ config:app:set context_chat auto_indexing --value=true --type=boolean
 
-* ``indexing_batch_size`` integer (default: 5000)
-   The number of files to index per run of the indexer background job (this is limited by `indexing_max_time`)
-
-.. code-block::
-
-   occ config:app:set context_chat indexing_batch_size --value=100 --type=integer
-
-* ``indexing_job_interval`` integer (default: 1800)
-   The interval at which the indexer jobs run in seconds
-
-.. code-block::
-
-   occ config:app:set context_chat indexing_job_interval --value=1800 --type=integer
-
-* ``indexing_max_time`` integer (default: 1800)
-   The number of seconds to index files for per run, regardless of batch size
-
-.. code-block::
-
-   occ config:app:set context_chat indexing_max_time --value=1800 --type=integer
-
-* ``request_timeout`` integer (default: 3000)
-   Request timeout in seconds for all requests made to the Context chat backend (the external app in AppAPI).
-   If a docker socket proxy is used, the ``TIMEOUT_SERVER`` environment variable should be set to a value higher than ``request_timeout``.
-
-.. code-block::
-
-   occ config:app:set context_chat request_timeout --value=3 --type=integer
```

Comment:
Removed docs for config options that we should make sure are good defaults out of the box. If we document them publicly, we risk that people change them without knowing what they're doing and we get support requests for them.